### DISCUSSION TITLE
Someone created an exact clone of GitHub!

### DISCUSSION THREAD

**Discussion by wrt54g** [community/community#32185] [Other Feature Feedback, Questions, & Ideas]
I found out that the website https://github.services.devops.takamol.support/ is an exact clone of GitHub. It also has all repositories and all user accounts on it.
EDIT: DO NOT PUT YOUR LOGIN DETAILS IN AS THIS MIGHT BE A SCAM!
![image](https://user-images.githubusercontent.com/85389871/188789181-c4bdf4c7-7408-4161-9cee-fbbe61a2768e.png)
I found it while I was checking backlinks to my website. 
Does anyone else know about this?

**Comment by radar-blip** [comment]
I'm sure whoever created it knows about it.
Anyway, you might want to let support and/or staff know about it: [Support Page](https://support.github.com/request)

**Comment by KalidasVijaybhak** [comment]
My antivirus prevented me from getting into the cloned site :)

**Reply by wrt54g** [reply]
@KalidasVijaybhak Yeah now I know why. When I enter fake login details, it redirects to an error page and it shows that error everytime. (I did not enter my real login) See:
![image](https://user-images.githubusercontent.com/85389871/188885026-12228d25-88ca-4e8d-abfb-007e3e9763ac.png)
In that case, this might be a scam website pretending to be GitHub.

**Reply by wrt54g** [reply]
It even shows the error if I leave both empty.

**Comment by harundemirkaya** [comment]
This could turn into a site to be used to pull information from GitHub users 🤣🤣

**Comment by viegasfh** [comment]
I found another one https://github.774.gs/.
